.circleci/config.yml

```diff
@@ -181,22 +181,13 @@ jobs:
       - checkout:
           path: ~/project
       - run:
-          name: "Download glyphs"
-          command: |
-            wget -O /tmp/glyphs.zip https://github.com/orangemug/font-glyphs/archive/gh-pages.zip
-            unzip /tmp/glyphs.zip
-            mv font-glyphs* font-glyphs
-            rm -f /tmp/glyphs.zip
-      - run:
-          name: "Build .deb"
-          command: ~/project/scripts/pack_deb.sh -v "0.$CIRCLE_BUILD_NUM" -m . -d "Mapbox styles.json with dependencies for the Ehrenamtskarte app" -n "eak-map-styles"
+          name: "Download legacy styles deb"
```

Comment:
Why do we download the old legacy styles here?

Reply:
If we just remove the legacy styles, then current installations will fail.